### DISCUSSION TITLE
As of last month, Github Issues is showing all the mails coming from the same user on Thunderbird (Windows only) and Safari

### DISCUSSION THREAD

**Discussion by ggarra13** [community/community#114142] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I use google mail and since a month ago, with Thunderbird on Windows (Linux is okay) and Safari on macOS all my emails I get from issues or discussions come from the same user (sfan).

Curiously, if I erase my thunderbird account and reconnect, the problem fixes itself until a new mail arrives into the mrv2 (my project folder).  The problem also happens on macOS.  But Linux is okay.  All my other folders and filters do not exhibit the issue on any of the OSes (Windows is fine there too).  The folder and emails are also fine in Gmail's web email.

The project is github.com:ggarra13/mrv2 if you guys want to know.

So it seems it is not Thunderbird nor Google mail nor OS.  Any clues what could be wrong?  It is driving me crazy.

**Comment by brianread108** [comment]
I've also got this problem with thunderbird on Linux (Mint22)

**Comment by ggarra13** [comment]
The problem is with Thunderbird.  Once you add a Contact that has the name of the list, all mails coming from a list or group will show as coming from the same user.

**Comment by brianread108** [comment]
Thanks, that fixed it.  No idea how that contact got added though!
